resources/views/auth/application/applications.blade.php

```diff
@@ -6,45 +6,52 @@
 
 @section('content')
     @if($workshops->count() > 1)
-        <form id="workshop-filter" method="GET" route="{{route('applications')}}">
-        <div class="card">
-            <div class="card-content">
-                <div class="row" style="margin-bottom: 0">
-
+    <div class="card">
+        <div class="card-content">
+            <div class="row center" style="margin-bottom: 0"> 
+                <form id="workshop-filter" method="GET" route="{{route('applications')}}">
                     <x-input.select id="workshop" :elements="$workshops" allow-empty :default="$workshop"
                                     text="Műhely"/>
                     @can('viewUnfinishedApplications', \App\models\User::class)
-                        <div class="col">
-                            @foreach (\App\Models\ApplicationForm::STATUSES as $st)
-                                <label>
-                                    <input type="radio" name="status" value="{{$st}}" @if($status == $st) checked @endif
-                                    ">
-                                    <span>@include('auth.application.status', ['status' => $st])</span>
-                                </label>
-                            @endforeach
-                        </div>
+                        @foreach (\App\Models\ApplicationForm::STATUSES as $st)
+                            <label>
+                                <input type="radio" name="status" value="{{$st}}" @if($status == $st) checked @endif>
+                                <span style="padding-left: 25px; margin: 5px">@include('auth.application.status', ['status' => $st])</span>
+                            </label>
+                        @endforeach
                     @endif
-                </div>
+                    <x-input.button type="submit" text="Szűrés"/>
+                </form>
+                <form id="empty-filter" method="GET" route="{{route('application')}}" style="{{($status=='' && $workshop=='')?'display: none':''}}">
+                    <x-input.button id="delete-filter" text="Szűrő törlése"/>
+                </form>
```

Comment:
Why don't you wrap this in an @if

Reply:
Because it needs to pop up if the workshop changes